test/evaluatorHelpers.test.ts

```diff
@@ -252,6 +252,24 @@ describe('renderPrompt', () => {
     expect(renderedPrompt).toBe('Test prompt value1');
     delete process.env.PROMPTFOO_DISABLE_TEMPLATING;
   });
+
+  it('should respect Nunjucks raw tags when variable is provided as a string', async () => {
+    const prompt = toPrompt('{% raw %}{{ var1 }}{% endraw %}');
+    const renderedPrompt = await renderPrompt(prompt, { var1: 'value1' }, {});
+    expect(renderedPrompt).toBe('{{ var1 }}');
+  });
```

Comment:
could also test the other escaping syntax:
`{{ '{{ var1}}' }}` => `{{ var1 }}`

Reply:
Is this valid escaping syntax?

Reply:
yes, both are valid https://jinja.palletsprojects.com/en/latest/templates/#escaping